tests/integration/test_transformers.py

```diff
@@ -0,0 +1,188 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from rdt import HyperTransformer
+from rdt.transformers import BaseTransformer
+from tests.datasets import BaseDatasetGenerator
+
+DATA_SIZE = 1000
+TEST_COL = 'test_col'
+
+# Mapping of rdt data type to dtype
+DATA_TYPE_TO_DTYPES = {
+    'boolean': ['b', 'O'],
+    'categorical': ['O', 'i', 'f'],
+    'datetime': ['M'],
+    'numerical': ['f', 'i'],
+    'integer': ['i'],
+    'float': ['f', 'i'],
```

Comment:
Right now, `OneHotEncodingTransformer` sometimes returns ints, but has an output type of `float`. I included `i` in this list to get the tests to pass. Should we update the output type of the transformer to `numerical`?

Reply:
Opened https://github.com/sdv-dev/RDT/issues/268